doc/source/conf.py

```diff
@@ -1,44 +1,29 @@
-# -*- coding: utf-8 -*-
-
 """Configuration file for the Sphinx documentation builder."""
 
-# This file does only contain a selection of the most common options. For a
-# full list see the documentation:
-# http://www.sphinx-doc.org/en/master/config
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
 
 # -- Path setup --------------------------------------------------------------
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
-import os
-import sys
-
 from pkg_resources import get_distribution
 
-# add the root path to path for mkdocs2sphinx docs
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..")))
-
 # these pages will have the version overwritten so that it doesn't fail
 # when a new version is released.
 REGRESSION_TEST_PAGENAMES = {"regression"}
 
 # -- Project information -----------------------------------------------------
 
 project = u"sphinx-bluebrain-theme"
-author = u"BlueBrain NSE"
-
-# The full version, including alpha/beta/rc tags
-release = get_distribution("sphinx-bluebrain-theme").version
-version = release
+version = get_distribution("sphinx-bluebrain-theme").version
+release = version
```

Comment:
this is the swap we talked about yesterday ?

Reply:
No this is a different thing, only `version` is used but it is best practice to define define them both. Once I merge this I'll do a release so a stable docs version is produced, which is what we discussed yesterday.

Reply:
ok